src/main/groovy/org/transmartproject/batch/batchartifacts/MultipleItemsLineItemReader.groovy

```diff
@@ -0,0 +1,103 @@
+package org.transmartproject.batch.batchartifacts
+
+import com.google.common.collect.Lists
+import org.springframework.batch.item.*
+import org.springframework.batch.item.file.FlatFileItemReader
+import org.springframework.batch.item.file.LineCallbackHandler
+import org.springframework.batch.item.file.ResourceAwareItemReaderItemStream
+import org.springframework.batch.item.file.mapping.DefaultLineMapper
+import org.springframework.batch.item.file.mapping.PassThroughFieldSetMapper
+import org.springframework.batch.item.file.separator.DefaultRecordSeparatorPolicy
+import org.springframework.batch.item.file.transform.DelimitedLineTokenizer
+import org.springframework.batch.item.file.transform.FieldSet
+import org.springframework.batch.item.validator.ValidationException
+import org.springframework.beans.factory.InitializingBean
+import org.springframework.core.io.Resource
+import org.springframework.util.Assert
+import org.springframework.util.ClassUtils
+
+import static org.springframework.batch.item.file.transform.DelimitedLineTokenizer.DELIMITER_TAB
+
+/**
+ * Reads wide file format where each row contains normally more than one item.
+ */
+class MultipleItemsLineItemReader<T> extends ItemStreamSupport
+        implements ResourceAwareItemReaderItemStream<T>, InitializingBean, Closeable {
+
+    {
+        setName(ClassUtils.getShortName(getClass()))
+    }
+
+    private static final String SAVED_CACHE = 'savedCache'
+    Resource resource
+    MultipleItemsFieldSetMapper<T> multipleItemsFieldSetMapper
```

Comment:
It should be clear what fields are to be specified from the outside. Instead of have them grouped with a private constant, with no comment.